tools/generate_svg_images.js

```diff
@@ -52,14 +20,12 @@ VF.Test.RUN_CANVAS_TESTS = false;
 VF.Test.RUN_SVG_TESTS = false;
 VF.Test.RUN_RAPHAEL_TESTS = false;
 VF.Test.RUN_NODE_TESTS = true;
-VF.Test.NODE_IMAGEDIR = path.resolve(__dirname, '..', 'build', 'images');
```

Comment:
I think we need to keep this line, because it allows the script to be run from anywhere.